templates/404.html

```diff
@@ -0,0 +1,16 @@
+{% extends "index.html" %}
+
+{% block title %}
+404
+{% endblock title %}
+
+{% block header_menu %}
+{{ menu_macros::menu_for(config=config, current_item="") }}
+{% endblock header_menu %}
+
+{% block content %}
+<div class="post">
+  <h1 class="post-title">Lost?</h1>
+  <p>This page does not exist.</p>
```

Comment:
It would be nice to have these configurable.

Reply:
Sure, I've added a few blocks around the heading/message!

(Making these configurable via toml felt a bit heavy handed, but let me know if you have alternative suggestions, naming or otherwise)

Reply:
Having named blocks is enough, thanks!